Zend/tests/readonly_props/variation.phpt

```diff
@@ -29,7 +29,7 @@ class Test {
         echo 'done';
     }
 
-    public function is() {
+    public function i() {
```

Comment:
Hm.  Disallowing `is` as a method name might cause issues and/or pushback for BC.  Have we checked to see how common that is?

Reply:
Right, I was able to avoid this BC break.